spec/did-method-spec-template.html

```diff
@@ -0,0 +1,456 @@
+            subnamespace  = ALPHA *(ALPHA / DIGIT / "_" / "-")<br>
+            char          = "1" / "2" / "3" / "4" / "5" / "6" / "7" / "8" / "9" / "A" / "B" / "C"<br/>
+              &nbsp;&nbsp;&nbsp;&nbsp;/ "D" / "E" / "F" / "G" / "H" / "J" / "K" / "L" / "M" / "N" / "P" / "Q"<br/>
+              &nbsp;&nbsp;&nbsp;&nbsp;/ "R" / "S" / "T" / "U" / "V" / "W" / "X" / "Y" / "Z" / "a" / "b" / "c"<br/>
+              &nbsp;&nbsp;&nbsp;&nbsp;/ "d" / "e" / "f" / "g" / "h" / "i" / "j" / "k" / "m" / "n" / "o" / "p"<br/>
+              &nbsp;&nbsp;&nbsp;&nbsp;/ "q" / "r" / "s" / "t" / "u" / "v" / "w" / "x" / "y" / "z"<br>
+          </code><br>
+          All Sovrin DIDs are base58 encoded using the Bitcoin/IPFS alphabets of a 16-byte uuid.
+          The encoding uses most alphas and digits, omitting 0OIl to avoid readability problems. This
+            gives an NSI length of either 21 or 22 characters, and it means that DIDs are case-sensitive and may not
+            be case-normalized, even though the prefix is always lower-case.<br>
+          <br>
+          Optional one or more sub namespaces may be specified to indicate which Sovrin system the DID should reference.
+          </p>
+        <h3>Namestring Generation Method</h3>
+        <p>The 16-byte uuid underlying a Sovrin DID can be generated in various ways--using standard uuid methods, for example, or
+            by selecting the first 16 bytes of a 256 bit Ed25519 verification key (the public
+          portion of the key pair). In the latter case, there may or may not be an active verification
+          key for the identity owner; that information is only available in the
+          key descriptions in the owner section of the DID Document.
+        </p>
+        <p>A convenient regex to match <code>sov</code> DIDs is: <br><br>
+            <code>^[1-9A-HJ-NP-Za-km-z]{21,22}$</code><br><br>
+           A convenient regex to match the entire did string is:<br><br>
+            <code>^did:sov:[1-9A-HJ-NP-Za-km-z]{21,22}(?&lt;namespace&gt;(?::\w[-\w]*)*)$</code>
+        </p>
+        <h3>Examples</h3>
+        <p>A valid <code>sov</code> DID might be: <code>did:sov:2wJPyULfLLnYTEFYzByfUR</code>.</p>
+    </section>
+    <section>
+      <h2>JSON-LD Context Definition</h2>
+        <p>On Sovrin, public DIDs and their corresponding documents are stored on the ledger.</p>
+       <p class="replace-me">TODO</p>
+    </section>
+    <section>
+        <h2>CRUD Operation Definitions</h2>
+
+        <h3>Create (Register)</h3>
+        <p>The DID method specification must specify how a client creates a DID record—the combination of a DID and its associated document—on the target system, including all cryptographic operations necessary to establish proof of ownership.</p>
+        <p>For example:</p>
+        <p>To create a DID, you must submit a transaction that looks like this: <br>
+          <pre>
+                {
+                    "submitterId": &lt;Trust Anchor DID&gt;,
```

Comment:
I think it's worth mentioning that the exact format of Write and Read requests may change. The essential part here is what data must be present in the requests (while the field names and format of the message is a subject to change).